app/views/candidate_interface/course_choices/review_enic_interruption/_waiting_maybe.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-three-quarters">
+    <div class="app-grid-column--grey">
+      <h1 class="govuk-heading-l"><%= t('review_enic_interruption.title') %></h1>
+
+      <p class="govuk-body"><%= t('review_enic_interruption.waiting_maybe.planned_link_html', link: govuk_link_to(t('review_enic_interruption.waiting_maybe.enic_link_text'), t('service_name.enic.statement_of_comparability_url'))) %></p>
```

Comment:
```suggestion
      <p class="govuk-body"><%= t('review_enic_interruption.waiting_maybe.planned_link_html', link: govuk_link_to(t('review_enic_interruption.waiting_maybe.enic_link_text', new_tab: true), t('service_name.enic.statement_of_comparability_url'))) %></p>
```

The text says we are opening this in a new tab, but it doesn't open in a new tab.